app/assets/javascripts/oxalis/view/layouting/golden_layout_adapter.js

```diff
@@ -66,7 +62,11 @@ const updateSizeForGl = gl => {
   if (!container) {
     return;
   }
-  const { width, height } = getGroundTruthLayoutRect();
+  let { width, height } = getGroundTruthLayoutRect();
```

Comment:
I'd create a function "castOptionalExtent" which takes width and height and turns them into 500 if they are undefined. So that you can write: `const { width, height } = castOptionalExtent(getGroundTruthLayoutRect());`. This pattern can be used at the multiple places, too.

Reply:
The current version of `getGroundTruthLayoutRect`guarantees to return a valid `width` and `height` (that is 500 when every way of determining the bounds failed). So there is no need to handle this anymore. I think I had already removed the checks like in line 52, but I think I was wrong ...